src/Queen.py

```diff
@@ -14,20 +14,30 @@
 
 
 class Queen(Piece):
-
     stringRep = 'Q'
     value = 9
 
-    def __init__(self, board: Board, side: bool, position: C, movesMade: int = 0):
+    def __init__(
+        self, board: Board, side: bool, position: C, movesMade: int = 0
+    ):
         super(Queen, self).__init__(board, side, position)
         self.movesMade = movesMade
 
     def getPossibleMoves(self) -> Iterator[Move]:
         currentPosition = self.position
 
-        directions = [C(0, 1), C(0, -1), C(1, 0), C(-1, 0), C(1, 1),
-                      C(1, -1), C(-1, 1), C(-1, -1)]
+        directions = [
+            C(0, 1),
+            C(0, -1),
+            C(1, 0),
+            C(-1, 0),
+            C(1, 1),
+            C(1, -1),
+            C(-1, 1),
+            C(-1, -1),
+        ]
         for direction in directions:
-            for move in self.movesInDirectionFromPos(currentPosition,
-                                                     direction, self.side):
+            for move in self.movesInDirectionFromPos(
+                currentPosition, direction, self.side
+            ):
```

Comment:
I guess this is debatable but I don't have a better option to offer, so it's ok.

Reply:
It's all debatable, that's the point here :) It's also why I just stick to `black` normally and don't bother, but it's just me.